lib/rubocop/cop/rspec/context_wording.rb

```diff
@@ -66,6 +67,12 @@ def joined_prefixes
         def prefixes
           cop_config['Prefixes'] || []
         end
+
+        def prefix_regex
+          escaped_prefixes =
+            prefixes.map { |prefix| "#{Regexp.escape(prefix)}\\b" }
+          /^#{escaped_prefixes.join('|')}/
```

Comment:
Don't ignore your intuition, @pirj. `Regexp.union` indeed seems to do escaping, and it adds a `(?: … )` group around the result, so the `/\b/` can be moved to the last line.
```suggestion
          /^#{Regexp.union(prefixes)}\b/
```

Reply:
Thanks for the suggestion, @bquorning. I had tried to do something similar but failed because I kept the word separator (`\b`) within the escaped prefixes before calling `union`. Yours is much cleaner. Thanks!

Reply:
> Don't ignore your intuition

That's my primary development driver 😄